src/components/Welcome.js

```diff
@@ -35,30 +51,53 @@ export default function Welcome() {
   };
 
   return (
-    <div>
-      <h1>Welcome to your Smart Shopping List!</h1>
+    <Box bg="brand.600">
+      <Box textStyle="roundedCorners">
+        <Link as={RouterLink} to="/new-list">
+          <AddIcon textStyle="AddIcon" />
+          <Text>Create New List</Text>
+        </Link>
 
-      {/* For accessibility reasons this is a link, but can be styled as button */}
+        <Text my="3%"> - or - </Text>
 
-      <Link to="/new-list">Create a new List</Link>
+        <Text>
+          Join an existing shopping list by entering a three word token
+        </Text>
 
-      <p>- or -</p>
-
-      <p>Join an existing shopping list by entering a three word token</p>
-      <form onSubmit={handleJoinList}>
-        <label htmlFor="tokenField"> Share Token </label>
-        <input
-          className={error ? 'error' : ''}
-          id="tokenField"
-          placeholder="three word token"
-          type="text"
-          aria-label="Enter your three word token"
-          value={userInputToken}
-          onChange={handleChange}
-        ></input>
-        {errorMessage ? <p className="error">{errorMessage}</p> : null}
-        <button type="submit">Join an existing list</button>
-      </form>
-    </div>
+        <FormControl>
+          <form onSubmit={handleJoinList}>
+            <VStack spacing="3%">
+              <VisuallyHidden>
+                <FormLabel htmlFor="tokenField"> Enter Token </FormLabel>
+              </VisuallyHidden>
+              <Input
+                variant="flushed"
+                w="30%"
+                textAlign="center"
+                textStyle={error ? 'error' : ''}
+                id="tokenField"
+                placeholder="three word token"
```

Comment:
Maybe I am too OCD, but can the first word start with a capital ? "Three word token" 

Although I think I would just refer to it as a token, we shouldn't give too much details about what it is like :)

Reply:
> Hi! Everything looks good. Just curious what the @emotion dependencies are? We don't seem to use them in the app. :)

Looking at the docs, the emotion dependencies seem to be part of the default install (https://chakra-ui.com/docs/getting-started)